Assets/CaseFile/Project.cs

```diff
@@ -25,7 +25,8 @@ private void InitializeData()
             PatientData = new Patient();
```

Comment:
I think we have missed the point of the Patient Object.

I'm not against there being a Project data type, but as per the initial Powerpoint - the patient object was initially specified to be:  

_This patient object should have accessible (or serve as an access point for) everything patient related.  
Patient information (Name/DOB/etc)
Components – and relevant transforms
Patient bone – and relevant transforms 
Landmarks_

At the moment, this is not the case.  it appears largely the same as existing IPS, with the Patient Data merely serving as text-based information.   Can we please move the current functionality of the project object into the patient object?

The **project** data type should be a place where we handle plans - implant choices, saved plans - any information added that doesn't explicitly describe the patient, but I definitely want the patient object to contain the mesh geometry, and the landmarks.

Reply:
@RemyaTP

Reply:
@KICO-SOURCE Meshes and Landmarks are moved to Patient class, the component information is as part of the plan, so that is kept there, in the project class.
Introduced a new class to load and parse the case file **CaseFileLoader**, so that the project class also very thin, also we could just remove **CaseFileLoader** class if we get rid of the case file in the future.